bluesky_database/frontend/src/app/__tests__/csv-export.test.tsx

```
@@ -0,0 +1,513 @@
+import '@testing-library/jest-dom'
+import { exportToCSV, escapeCSVField, type Post } from '@/utils/csvExport'
+
+
+// Mock data structure matching the page component
+interface Post {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Post'. Consider to delete it or rename it.

'Post' is defined here:


(lint/suspicious/noRedeclare)